Libraries/xcsdk/Tests/test_PlatformVersion.cpp

```diff
@@ -37,8 +37,8 @@ TEST(PlatformVersion, Parse)
     auto platformVersion = PlatformVersion::Open(&filesystem, "/" + platform);
```

Comment:
In general, I think it's too much work and not too valuable to test parsing each key in each type of thing that's added. There's thousands of keys and it's just checking that they're written right in the original file. My current plan is to add tests for any keys that need *changes* to their parsing, usually from a bug.